src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EditProfile.razor

```diff
@@ -58,7 +58,7 @@
                 <BitLabel>Full name</BitLabel>
                 <BitTextField @bind-Value="UserToEdit.FullName"
                           Placeholder="Full name"
-                          OnKeyUp="WrapHandled(CheckSaveButtonEnabled)" />
+                          OnChange="WrapHandled(CheckSaveButtonEnabled)" />
```

Comment:
What happened here?

Reply:
If you copy a value and then paste it to the BitTextField using mouse it was not going to change the enable status of it. OnChange solves the issue. @ysmoradi

Reply:
is it going to handle every scenario? plz confirm it with @maryamhdr and @sajadkardel 
anyway, this change is not related to this issue, so plz remove it and submit a new issue if required.

Reply:
It handles all scenarios, however, I removed it from the PR, and I'll follow it with @maryamhdr. @msynk @ysmoradi